MutationObserver.js

```diff
@@ -0,0 +1,349 @@
+// https://chromium.googlesource.com/external/github.com/dart-lang/web-components/+/refs/tags/v0.12.3/lib/MutationObserver.js
```

Comment:
TODO can we remove this in favour of a simpler interval function that clears itself ?